test/WormholeSender.t.sol

```diff
@@ -61,3 +61,10 @@ contract QuoteDeliveryCost is WormholeSenderTest {
     assertGt(cost, 0, "No cost was quoted");
   }
 }
+
+contract UpdateGasLimit is WormholeSenderTest {
+  function testFuzz_CorrectlyUpdateGasLimit(uint256 gasLimit) public {
+    wormholeSender.updateGasLimit(gasLimit);
+    assertEq(wormholeSender.GAS_LIMIT(), gasLimit, "The gas limit is incorrect");
```

Comment:
can we add an expectEmit, and also add a test to make sure _notOwner can't call this method?

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/94/commits/5fb57cdad6d3b0bce2cf8f63c6c873ef27f761ae